src/Plugins/RpcServer/RpcServer.Blockchain.cs

```diff
@@ -156,8 +203,16 @@ protected virtual JToken GetRawMemPool(JArray _params)
             return json;
         }
 
+        /// <summary>
+        /// Gets a transaction by its hash.
+        /// </summary>
+        /// <param name="_params">
+        /// An array containing the transaction hash as the first element,
+        /// and an optional boolean indicating whether to return verbose information.
+        /// </param>
+        /// <returns>The transaction data as a <see cref="JToken"/>. In json format if the second item of _params is true, otherwise base64string. </returns>
```

Comment:
I see the `base64string` pattern being repeated, so it's probably intentionally.